app/css/buddyup.css

```diff
@@ -29,6 +31,21 @@ html {
 .no-data {
     margin: 1rem;
 }
+.user-list ul {
```

Comment:
Please move this into a user-list.css file. Also, using flexbox here would make sense. You wouldn't have to specify the height twice.